kenneth051/mydiary-react
# set up a new project

**What does this PR do?**
sets up new react project
**Description of the task to be completed**
-This PR creates a new react project
-creates a nav bar to show the project works
-introduces bootstraps as a CSS styling framework to be used
**How should this be tested manually**
git pull, check out into this branch
run `npm install` to install all dependencies
run `npm start` to run the server

**Relevant Pivotal Tracker Stories**

[#162230638](https://www.pivotaltracker.com/n/projects/2228292/stories/162230638)

#### Any background context you want to add?
N/A
**screen shot**
<img width="760" alt="screen shot 2018-11-27 at 17 46 42" src="https://user-images.githubusercontent.com/40117122/49089489-b9a83b00-f26c-11e8-9c12-c21a28aa1fed.png">
